MoreLinq.Test/Enumerable.cs

```diff
@@ -0,0 +1,538 @@
+namespace MoreLinq.Test
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using LinqEnumerable = System.Linq.Enumerable;
+
+    [DebuggerStepThrough]
+    static partial class Enumerable
```

Comment:
Do we really need all these methods? Or did you just put all non-conflicting methods here?

Reply:
How do you define those as _we actually need_? Are all the methods in there used? No, so no they're not needed _today_? The idea, however, was to save trouble for a future contributor.

Reply:
It's not really a big deal, but I don't really like dead/unused code. It's not hard to add the forward either so they can be added as needed. I don't feel strongly about this so lets keep all forwards if you prefer that.